test_gpt2.cu

```diff
@@ -30,10 +31,10 @@ int main(int argc, char *argv[]) {
     GPT2 model;
     gpt2_build_from_checkpoint(&model, "gpt2_124M.bin");
 
-    int C = model.config.channels;
+    //int C = model.config.channels;
```

Comment:
The following warning I got on Windows CUDA:

```
`warning #177-D: variable "<var>" was declared but never referenced`
```
Only commented all these out instead of removing in case you still want them for something